Mathlib/RingTheory/Ideal/Maps.lean

```diff
@@ -1067,6 +1067,12 @@ lemma comap_ker {C : Type*} [Semiring C] [Algebra R C] (f : B →ₐ[R] C) (g :
     (RingHom.ker f).comap g = RingHom.ker (f.comp g) :=
   RingHom.comap_ker f.toRingHom g.toRingHom
 
+/-- Synonym for `RingHom.ker_coe_equiv`, but given an algebra equivalence. -/
+@[simp] theorem ker_coe_equiv {R A B : Type*} [CommSemiring R] [Ring A] [Ring B]
```

Comment:
It seems odd that this is in the AlgHom namespace with no AlgHom in sight, but I guess there's an argument to be made that it's consistent with the other variants.